src/allmydata/test/common.py

```diff
@@ -958,12 +961,12 @@ def _corrupt_offset_of_block_hashes_to_truncate_crypttext_hashes(data, debug=Fal
     assert sharevernum in (1, 2), "This test is designed to corrupt immutable shares of v1 or v2 in specific ways."
     if sharevernum == 1:
         curval = struct.unpack(">L", data[0x0c+0x18:0x0c+0x18+4])[0]
-        newval = random.randrange(0, max(1, (curval/hashutil.CRYPTO_VAL_SIZE)/2))*hashutil.CRYPTO_VAL_SIZE
+        newval = random.randrange(0, max(1, (curval/hashutil.CRYPTO_VAL_SIZE)//2))*hashutil.CRYPTO_VAL_SIZE
```

Comment:
There's an additional `/` that didn't get converted to `//`, likewise below.

Reply:
I figured the float division was fine in that case, but I can agree to preserving the logic exactly. Done in 3ae927eab951fbb7f28cc85e8c2247f906a9674c.